File: sdkwork-app-sdk-csharp/Models/VipPackGroupQueryForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class VipPackGroupQueryForm
    {
        public int? VipLevelId { get; set; }
        public bool? RecommendedOnly { get; set; }
    }
}

File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/VipPackGroupQueryForm.java
package com.sdkwork.app.model;

public class VipPackGroupQueryForm {
    private Integer vipLevelId;
    private Boolean recommendedOnly;

    public Integer getVipLevelId() {
        return this.vipLevelId;
    }
    
    public void setVipLevelId(Integer vipLevelId) {
        this.vipLevelId = vipLevelId;
    }

    public Boolean getRecommendedOnly() {
        return this.recommendedOnly;
    }
    
    public void setRecommendedOnly(Boolean recommendedOnly) {
        this.recommendedOnly = recommendedOnly;
    }
}
